cmd/otelcol/config/collector/otlp_config_linux.yaml

```diff
@@ -117,7 +118,7 @@ service:
     metrics:
       receivers: [otlp, signalfx, smartagent/signalfx-forwarder]
       processors: [memory_limiter, batch]
-      exporters: [signalfx]
+      exporters: [otlphttp, signalfx]
```

Comment:
The question is if here we need both exporters. Same for traces.

Reply:
The signalfx exporter has a number of built in translations that work best w/ existing built-in content. You are also now adding double publishes.